src/handbook/customer/marketing/social-media.md

```diff
@@ -0,0 +1,97 @@
+|Product updates      |*        |*        |*        |         |        |*      |*             |*                 |
+|Curated content      |*        |*        |*        |         |        |       |              |                  |
+|Events               |*        |*        |*        |         |*       |*      |*             |*                 |
+|Short videos         |*        |*        |*        |*        |        |       |              |                  |
+|Long format videos   |         |*        |         |*        |        |       |              |                  |
+|Downloadable content |*        |*        |*        |         |*       |       |*             |                  |
+
+## Strategy for increasing reach
+
+- Share quality content aligned with the [company strategy](/handbook/company/strategy/) and of interest to our [personas](/handbook/product/personas/#personas) and [ideal customer profile](https://docs.google.com/document/d/1krMIUJvosw8xUQog_iq_FEvI9R5WEo9ZyWUCdTb9XnQ/edit#heading=h.3rr2wuny55dl). 
+- Position ourselves as thought leaders.
+- Adapt content frequency and type to each channel.
+- Be social:
+    - Respond promptly to comments and messages.
+    - Like and share relevant content from other sources.
+- Leverage trending topics and holidays.
+
+### Scope
+
+- Channels: LinkedIn, Facebook, Twitter, Youtube.
+
+### Tactics
+
+#### Frequency
+
+- Once a work day.
+
+#### Type of content
+
+- **Own content:** 3x week. Mondays, Wednesdays, and Thursdays.
+    - 2 fresh blog articles: shared as a link to the article. The target day for review should be 3 work days before posting date, and the tile image should be requested when it's submited for review.
+    - 1 repurposed blog article: in the form of slides with summary of the article, diagram, new tile image, image with a quote,animation or short video.
+    - Product updates: this would replace one of the previous type of content, it could be share as a short video, short tutorial,animation, diagram.
+- **Curated content from other sources:** 1x week. Fridays.
+    - Articles relevant to our ideal customer / industry related / news / posts.
+        - Avoid competitor mentions.
+        - Must be industry-related and of interest to our audience.
+        - Choose reliable sources.
+        - Fresh, up-to-date, and informative.
+        - Include our POV in the post.
+        - Tag the original source.
+    - Follow partners and customers on each channel for shareable content..
+- **Other content types:** 1x week. Tuesdays.
+    - Quick tips, repurposed YouTube videos, surveys, product updates, events, downloads, FAQs, best practices, testimonials, quizzes,challenges.
+- [Detailed list here](https://docs.google.com/spreadsheets/d/1Ifz6f7D_ZxCxJkAS9LbdvdnEAae0tGFEqkGVHcnn2TU/edit?usp=sharing).
+
+#### Planning and posting
+
+- We'll plan **two weeks ahead**, to give us time to prepare content and the associated materials.
```

Comment:
While there's a plan, how do we remain open to changes to that plan?

Reply:
I added a final point mentioning how the plan is flexible